### PR TITLE
Minor fixes to the HidInjectorSample

### PR DESCRIPTION
**Pull request by ooeygui**
This change fixes some potential collisions with the hid injector in the maker images.

### CODE REVIEW

File: HidInjector/driver/HidInjectorKd.c [hunk 1]
```diff
@@ -287,7 +288,8 @@ Return Value:
 	// Initialize VHF.  This will talk to HIDCLASS for us.
 	//
 	status = HIDINJECTOR_VhfInitialize(device);
-	if (!NT_SUCCESS(status)) {
+	if (!NT_SUCCESS(status)) 
+    {
```
Comment (alanch-ms) [inline HidInjector/driver/HidInjectorKd.c:295]: Can you check the indents (ie. space vs tabs?)  On chrome, the indents are off, but they look fine in IE.  Not sure which one is correct.

File: HidInjector/driver/HidInjectorKd.c [hunk 2]
```diff
@@ -296,17 +298,18 @@ Return Value:
 	// Create a new queue to handle IOCTLs that will be forwarded to us from
 	// the rawPDO. 
 	//
-	status = RawQueueCreate(
-		device,
-		&deviceContext->RawPdoQueue);
-	if (!NT_SUCCESS(status)) {
+	status = RawQueueCreate(device, &deviceContext->RawPdoQueue);
+	if (!NT_SUCCESS(status)) 
+    {
```
Comment (alanch-ms) [inline HidInjector/driver/HidInjectorKd.c:306]: Ditto - indent

File: HidInjector/driver/HidInjectorKd.c [hunk 3]
```diff
@@ -296,17 +298,18 @@ Return Value:
...
 		KdPrint(("QueueCreate failed 0x%x\n", status));
 		return status;
 	}
 
-	if (NT_SUCCESS(status)) {
+	if (NT_SUCCESS(status)) 
+    {
```
Comment (alanch-ms) [inline HidInjector/driver/HidInjectorKd.c:312]: Ditto - indent

File: HidInjector/driver/HidInjectorKd.c [hunk 4]
```diff
@@ -296,17 +298,18 @@ Return Value:
...
 		status = HIDINJECTOR_CreateRawPdo(device);
-		if (!NT_SUCCESS(status)) {
+		if (!NT_SUCCESS(status)) 
+        {
```
Comment (alanch-ms) [inline HidInjector/driver/HidInjectorKd.c:315]: Ditto - indent

**Comment by alanch-ms** [comment]
Minor comments.  Otherwise :+1:

File: HidInjector/driver/HidInjectorKd.c (hunk 1 above)
Comment (ooeygui) [inline HidInjector/driver/HidInjectorKd.c:295]: Thanks for finding this.

**Comment by ooeygui** [comment]
The original file was tabs, I was configured for spaces. Fixed.

**Pull request merged**